### ISSUE TITLE
Windows: TypeError: no implicit conversion of nil into String

### ISSUE THREAD

**Issue by webframp** [hw-labs/batali#81]
`batali resolve` is successful without issue but `install` fails with the TypeError. Purging the cache is an ok fix. Seems to be reproducible as needed. 

```
batali - Cookbook Collection Manager - [Version: 0.4.8]
- I got 99 problems but a policyfile ain't one
```

```
C:\src\infra-repo [master ≡ +1 ~1 -0 !]> be batali install --debug
[Batali]: Readying installation destination... complete!
[Batali]: Installing cookbooks... [DEBUG]: Cache directory to persist cookbooks: C:/Users/SESCRIVA/.batali/cache
[DEBUG]: Cache directory to persist cookbooks: C:/Users/SESCRIVA/.batali/cache
[DEBUG]: Cache directory to persist cookbooks: C:/Users/SESCRIVA/.batali/cache
[DEBUG]: Cache directory to persist cookbooks: C:/Users/SESCRIVA/.batali/cache
[DEBUG]: Cache directory to persist cookbooks: C:/Users/SESCRIVA/.batali/cache
[DEBUG]: Cache directory to persist cookbooks: C:/Users/SESCRIVA/.batali/cache
[DEBUG]: Cache directory to persist cookbooks: C:/Users/SESCRIVA/.batali/cache
[DEBUG]: Cache directory to persist cookbooks: C:/Users/SESCRIVA/.batali/cache
[DEBUG]: Cache directory to persist cookbooks: C:/Users/SESCRIVA/.batali/cache
[DEBUG]: Cache directory to persist cookbooks: C:/Users/SESCRIVA/.batali/cache
error!
[ERROR]: Reason - no implicit conversion of nil into String
ERROR: TypeError: no implicit conversion of nil into String

C:\src\infra-repo [master ≡ +1 ~1 -0 !]> be batali cache --scrub
[Batali]: Remove all contents from local cache (C:/Users/SESCRIVA/.batali/cache) (Y/N): Y
[Batali]: Scrubbing local cache... complete!
Batali cache information:
  Path: C:/Users/SESCRIVA/.batali/cache
  Size: 0.00M

C:\src\infra-repo [master ≡ +1 ~1 -0 !]> be batali install
[Batali]: Readying installation destination... complete!
[Batali]: Installing cookbooks... complete!
```

**Comment by webframp** [comment]
install with `DEBUG=true`:

```
[Batali]: Readying installation destination... complete!
[Batali]: Installing cookbooks... error!
[ERROR]: Reason - no implicit conversion of nil into String
ERROR: TypeError: no implicit conversion of nil into String
C:/Users/SESCRIVA/AppData/Local/chefdk/gem/ruby/2.1.0/gems/batali-0.4.8/lib/batali/source/site.rb:101:in `dirname'
C:/Users/SESCRIVA/AppData/Local/chefdk/gem/ruby/2.1.0/gems/batali-0.4.8/lib/batali/source/site.rb:101:in `clean_asset'
C:/Users/SESCRIVA/AppData/Local/chefdk/gem/ruby/2.1.0/gems/batali-0.4.8/lib/batali/command/install.rb:42:in `ensure in block (5 levels) in execute!'
C:/Users/SESCRIVA/AppData/Local/chefdk/gem/ruby/2.1.0/gems/batali-0.4.8/lib/batali/command/install.rb:42:in `block (5 levels) in execute!'
```

**Comment by chrisroberts** [comment]
Fixed in v0.4.10. Thanks!
